docs/reference/sql/endpoints/jdbc.asciidoc

```diff
@@ -148,6 +148,13 @@ will be - typically the first in natural ascending order) for fields with multip
 ==== Index
 `index.include.frozen` (default `false`):: Whether to include frozen indices in the query execution or not (default).
 
+[discrete]
+==== Cluster
+`catalog`:: Default catalog (cluster) for queries. If unspecified, the
+queries execute on the data in the local cluster only.
++
+experimental:[] See <<modules-cross-cluster-search, cross cluster search>>.
```

Comment:
```suggestion
experimental:[] See <<modules-cross-cluster-search,{ccs}>>.
```

It should be `cross-cluster search` with a hyphen. However, you can use the `{ccs}`, which auto-expands to that text.